spec/models/note_spec.rb

```diff
@@ -1,6 +1,82 @@
 require 'rails_helper'
 
+shared_examples 'a content_length response' do |length|
+  it "returns #{length}" do
+    expect(note_with_attributes.content_length).to eq(length)
+  end
+end
+
 RSpec.describe Note, type: :model do
-  it do
+  subject(:note) { create(:note) }
+
+  it { is_expected.to belong_to(:user) }
+
+  it 'has a valid factory' do
+    expect(subject).to be_valid
+  end
+
+  %i[user_id title content note_type].each do |value|
+    it { is_expected.to validate_presence_of(value) }
+  end
+
+  describe '#validate_review_word_limit' do
+    let(:utility) { create(%i[north_utility south_utility].sample) }
+    let(:user) { create(:user, utility: utility) }
+    let(:review_note) { build(:note, note_type: :review, user: user) }
+
+    before do
+      allow(review_note).to receive(:content_length).and_return(content_length)
+    end
+
+    context 'when content length is short' do
+      let(:content_length) { 'short' }
+
+      it 'does not have errors' do
+        review_note.save
```

Comment:
Existira una forma de no tener que repetir este save en cada it del contexto? :repeat:

Reply:
Si use el before que tenia para el stub y aproveche a usar la sentencia `note.save` ahi para que lo haga en cada it y no haga falta escribirlo en todos los it